### PR TITLE
elpi 1.4.1

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

**Comment by gares** [comment]
Linux failure is spurious (network error).
The OSX failure is like last week I guess #14325

File: packages/elpi/elpi.1.4.1/opam [hunk 1]
```diff
@@ -0,0 +1,82 @@
+opam-version: "2.0"
+name: "elpi"
```
Comment (mseri) [inline packages/elpi/elpi.1.4.1/opam]:
```suggestion
```

This field is redundant. Opam will override it with the name inferred from the path

**Comment by gares** [comment]
Thanks, I'll fix and push

File: packages/elpi/elpi.1.4.1/opam (hunk 1 above)
Comment (gares) [reply packages/elpi/elpi.1.4.1/opam]: done

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**